docs/system-overview.rst

```diff
@@ -0,0 +1,84 @@
+Proctoring System Overview
+===========================
+This document outlines the components involved in the edX proctoring system. It should
+serve as a guide to orient developers on what components exist, how they interact, and 
+where to find the source code in our platform.
+
+.. contents::
+
+System Components
+------------------
+
+.. image:: images/components.png
+
+Proctored Exam Views
+^^^^^^^^^^^^^^^^^^^^
+
+Interstitial views within a section that are shown to in place of the actual
+exam content. They are used to walk a leaner through setup steps
+and display the state of the current attempt when the exam is not in progress.
+
+The LMS will call into edx-proctoring to load the relevent template when rendering the
```

Comment:
This line is too verbose. How about
`During a proctored exam, LMS renders different student views based on existing exam status. The LMS renders the views by calling into edx-proctoring and load the relevent html templates`

Reply:
I usually like to say that the intestitials are "driven by" the status of an attempt; as the attempt's status gets updated, we choose which interstitials to show/whether to show an interstitial at all. I think Simon's idea of separating the ideas of 1. calling into the edx-proctoring plugin to do this, from 2. the status column driving the choice of interstitial is a good one.

With that said I think there are a lot of valid ways of phrasing this. Just trying to give perspective of ways I've explained it in the past that seemed helpful for folks (although maybe it was just helpful for me!)